client/src/main/scala/skuber/apps/Deployment.scala

```diff
@@ -1,14 +1,15 @@
-package skuber.ext
+package skuber.apps
 
 /**
  * @author David O'Riordan
  */
 
+import skuber.ResourceSpecification.{Names, Scope}
 import skuber._
 
 case class Deployment(
     val kind: String ="Deployment",
-    override val apiVersion: String = extensionsAPIVersion,
+    override val apiVersion: String = "apps/v1beta1",
```

Comment:
this doesn't seem to be backwards compatible with older versions of Kubernetes

Reply:
Yes, the original comment above (see Deployment) describes the impact and mitigation of that - at the moment Deployment only exists in the apps package and has the newer API version and the Deployment under ext is simply an alias to that - it would be easy to add back  the full Deployment class under ext package with the older API version if we want, and users can import whichever is appropriate for them and their version of k8s